styles/dropdown.css

```diff
@@ -0,0 +1,124 @@
+/*---
+section: Helpers
+title: Dropdown
+---
+
+#### Variables
+```
+  dropdown-bg: '#fff'
+```
+
+#### Example
+```example:html
+<div class="dropdown open">
+  <button class="button dropdown-toggle">Dropdown Button</button>
+  <div class="dropdown-menu">
+    <button class="dropdown-item">Button item</button>
+    <button class="dropdown-item">Button item</button>
+    <a href="" class="dropdown-item">Link item</a>
+    <a href="" class="dropdown-item">Link item</a>
+  </div>
+</div>
+```
+*/
+.dropdown {
+  display: inline-block;
+  position: relative;
+}
+
+.dropdown-toggle {
+  transition: all ease .5s;
+
+  &::after {
+    display: inline-block;
+    width: 1em;
+    height: 1em;
+    margin-left: .25rem;
+    vertical-align: middle;
+    // triangle mixin here
+    content: '';
+  }
+
+  &:focus {
+    outline: 0;
+  }
+}
+
+.dropdown-menu {
+  display: none;
+  position: absolute;
+  top: 100%;
+  left: 0;
+  z-index: 1000;
+  float: left;
+  min-width: 160px;
+  margin: 2px 0 0;
+  border: var(--border-size) solid var(--color-border);
+  border-radius: var(--border-radius);
+  padding: 0;
+	background-color: var(--dropdown-bg);
+  background-clip: padding-box;
+  font-size: var(--font-size-body);
+  text-align: left;
+  list-style: none;
```

Comment:
Dont' see a reason not, but if we have a better design that doesn't use it, I'm fine with that.